app/src/main/res/values-ca/arrays.xml

```diff
@@ -23,6 +23,6 @@
     </string-array>
     <string-array name="remote_languages">
         <item>Anglès</item>
-        <!--        <item>Espanyol (beta)</item>-->
+        <item>Espanyol (beta)</item>
```

Comment:
It seems that the API in Spanish is no longer available, so we should remove this option from the configuration and remote recipe search.

Reply:
I have commented this line again. In order to remove Spanish from available remote languages, we should create a new issue.